packages/ui/src/components/Card/Card.module.css

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+@layer tokens, base, components, utilities;
```

Comment:
Is this only for BUI CSS, or would it need to be used for custom styles in plugins as well?

I'm assuming not since you generally wouldn't use layers there.